backend/audit/test_models.py

```diff
@@ -23,7 +23,7 @@ class SingleAuditChecklistTests(TestCase):
     def test_str_is_id_and_uei(self):
         """String representation of SAC instance is #{ID} - UEI({UEI})"""
```

Comment:
```suggestion
    def test_str_is_id_report_id_and_uei(self):
        """String representation of SAC instance is #{ID}--{report_id}--{UEI}"""
```

Should we update these as well?

Reply:
Whoops, yes, I should have added that too.

Reply:
I'll add that in another PR.